### ISSUE TITLE
[Bug]: A11Y Bugs of Sbb-FlipCard

### ISSUE THREAD

**Issue by Mursel41** [sbb-design-systems/lyne-components#2983]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Accessibility

### It affects the following packages

Components

### Input mode

None

### What happened?

1. card heading only reachable backwards. If you reach Card via Tab, you cannot reach the "Summary" heading with the linearly arrow keys. 
2. card only has a nav hint but no label: aria-label="Click on this card for details" - card heading could be the label.
3. buttons in card (+ and x) are focused separately after the card, although they have no further functionality. They are also not labeled.

### Steps to reproduce the issue

xxx
